pkg/augerctl/ctl.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
I would lean towards aligning with structure of etcd `etcdctl` and `etcdutl` where commands are subdirectory of `cmd` rather than `pkg`, i.e:
- https://github.com/etcd-io/etcd/tree/main/etcdctl/ctlv3/command
- https://github.com/etcd-io/etcd/tree/main/etcdutl/etcdutl

cc @siyuanfoundation for second opinion before changing it.